src/Tests/IQsharpEngineTests.cs

```diff
@@ -261,6 +261,22 @@ public async Task UpdateSnippet()
             await AssertSimulate(engine, "HelloQ", "msg0", "msg1");
         }
 
+        [TestMethod]
+        public async Task DumpToFile()
+        {
+            var engine = Init();
+
+            // Compile DumpMachine snippet.
+            await AssertCompile(engine, SNIPPETS.DumpToFile, "DumpToFile");
+
+            // Run, which should produce files in working directory.
+            await AssertSimulate(engine, "DumpToFile", "Dumped to file!");
+
+            // Ensure the expected files got created.
+            Assert.IsTrue(System.IO.File.Exists("DumpMachine.txt"));
+            Assert.IsTrue(System.IO.File.Exists("DumpRegister.txt"));
```

Comment:
Minor suggestion - could you then delete the files after verifying that they exist?

Reply:
Sure thing, should be easy enough to add!